### ISSUE TITLE
[Android][WebView] injectedJavaScript don't work

### ISSUE THREAD

**Issue by retyui** [react-native-community/react-native-webview#40]
When I [debug `WebView`](https://bamtech.gitbook.io/dev-standards/react-native/debugging/debug-webviews.mo) 
I understand that after serialize (on andoid) from the script, delete all the line breaks, because of what you have to be very careful.

## Example:
```js
// it's work fine
const validScript = `(function(){
alert(123);
})()`;
<WebView injectedJavaScript={validScript} />
```

```js
// It's not work on Android
const invalidScript = `
(function(){
        alert(123); // some comment
})();`;
<WebView injectedJavaScript={invalidScript} />
```
### As a result, the  webview will get this code: (SyntaxError: Unexpected end of input)
```js
(function(){        alert(123); // some comment})();
```

And another problem may arise because of a forgotten semicolon:
```
(function(){
alert(1)
window.postMessage('123')
})();
```
### SyntaxError: Unexpected identifier
```js
(function(){alert(1)window.postMessage('123')})();
```

**Comment by jamonholmgren** [comment]
@retyui Thanks for the report! Do you have suggestions on how this might be fixed?

**Comment by retyui** [comment]
No

**Comment by Titozzz** [comment]
Should be fixed by #41
